Marlin/ubl_G29.cpp

```diff
@@ -1615,22 +1615,22 @@ double zzz[] = { 0.05, 0.05, 0.15, 0.15 };
 int three_size = sizeof(xxx) / sizeof(double);
 
       results = lsf_linear_fit(xxx9, yyy9, zzz9, nine_size);
-SERIAL_PROTOCOLPAIR("\nxxx9->A =", results->A); 
-SERIAL_PROTOCOLPAIR("\nxxx9->B =", results->B); 
-SERIAL_PROTOCOLPAIR("\nxxx9->D =", results->D); 
-SERIAL_PROTOCOL("\n");
+SERIAL_ECHOPAIR("\nxxx9->A =", results->A); 
+SERIAL_ECHOPAIR("\nxxx9->B =", results->B); 
+SERIAL_ECHOPAIR("\nxxx9->D =", results->D); 
+SERIAL_ECHO("\n");
```

Comment:
You've got my curiosity up.  Why is there both a SERIAL_ECHO and a SERIAL_PROTOCOL series?